gulpfile.js

```diff
@@ -70,6 +70,34 @@ const cssLoaderPlugin = {
 	},
 };
 
+gulp.task('ext:lint-staged', async (done) => {
+	const fix = (argv.fix === undefined) ? false : true;
+	const {stdout, stderr} = await exec("git diff --cached --name-only");
+	const stagedFiles = stdout.split('\n').filter(file => file.endsWith('.ts') || file.endsWith('.tsx'));
```

Comment:
.css as well?

Reply:
eslint doesn't work on css. I will have to get prettier running here somehow.